src/plugins/janus_sip.c

```diff
@@ -6383,6 +6384,7 @@ void janus_sip_sdp_process(janus_sip_session *session, janus_sdp *sdp, gboolean
 				session->media.has_audio = TRUE;
 				session->media.remote_audio_rtp_port = m->port;
 				session->media.remote_audio_rtcp_port = m->port+1;	/* FIXME We're assuming RTCP is on the next port */
+				session->media.dtmf_pt = janus_sdp_get_codec_pt(sdp, 0, "dtmf"); 
```

Comment:
You're assuming `0` will always be the m-index of audio, which is wrong: there's no guarantee the first m-line will be for audio, or that there will be audio at all. The proper way to use `janus_sdp_get_codec_pt` in those cases is by passing `-1`: since `dtmf` is associated to audio, it will make the SDP utils find the first audio m-line in the SDP and look there.